Clinical-Genomics/scout
# A faster-built and smaller Docker image

This PR should be reviewed after fixing #2967 

This PR:
- Reduces the time to build the Scout Dockerfile from 4,5 minutes to 70 seconds (fix #2908)
- Reduces the size of the Scout image from 764 Mb to 380 Mb 

**How to test**:
1. Remove all local scout-related containers and images
2. Clear the Docker cache: `docker builder prune`
2. from master branch, build scout image with the command: `docker build -t scout .`
3. Write down the time it takes and the final size of the the image
4. Switch to this branch
5. Repeat step 1
6. Make sure that the image works, by running `docker-compose up` and going to localhost:5000

**Expected outcome**:
- The image from this branch should compile faster and should be smaller in size than the one from master branch

**Review:**
